tests/statsmodels/test_statsmodels_model_export.py

```diff
@@ -107,42 +107,34 @@ def _test_model_save_load(statsmodels_model, model_path, *predict_args):
 
 
 def _test_model_log(statsmodels_model, model_path, *predict_args):
-    old_uri = mlflow.get_tracking_uri()
     model = statsmodels_model.model
     with TempDir(chdr=True, remove_on_exit=True) as tmp:
-        for should_start_run in [False, True]:

```

Comment:
The behavior tested by this case can be captured by simplifying to `should_start_run=False`.